flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/ExecutionCheckpointingOptions.java

```diff
@@ -159,4 +159,12 @@
 					"If during checkpointing, checkpoint start delay exceeds this timeout, alignment " +
 					"will timeout and checkpoint barrier will start working as unaligned checkpoint.")
 				.build());
+
+	public static final ConfigOption<Boolean> FORCE_UNALIGNED =
```

Comment:
You probably need to generate docs and add them to this commit.
` mvn install -Dgenerate-config-docs -pl flink-docs`

Reply:
Done.